recipes/rb-jekyll-default-layout/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "jekyll-default-layout" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 7d128fc8972353370cdf22beb1610434421c732d707ba4c77513d65941bb3f01
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
```

Comment:
no gem unpack needed? Interesting ...

Reply:
In this case I had to bundle the license with the recipe. It was not included in the gem.